storage/innobase/include/dict0mem.h

```diff
@@ -1552,7 +1606,7 @@ struct dict_instant_t
 	unsigned leaf_redundant:1;
 	/** Dropped columns */
 	dict_col_t* dropped;
-	/** Map of clustered index fields[i - first_user_field()]
+	/** Map of non-PK clustered index fields[i - first_user_field()]
```

Comment:
The word non-PK should be added before ‘fields’, because the suggested wording is ambiguous: it could misleadingly refer to ‘non-PK clustered index’, a.k.a. `dict_index_t::is_gen_clust()`.

Reply:
Fixed.